Components/card.css

```diff
@@ -0,0 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+.bat-card {
+    font-family: 'Poppins', sans-serif;
+    max-width: 300px;
+    overflow: hidden;
+    padding: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    gap: .25rem;
+    width: 100%;
+    position: relative;
+}
+
+.bat-card .bat-badge {
+    display: none;
+}
+
+.bat-card.card-with-badge .bat-badge {
+    padding: 0.25rem 0.5rem;
+    font-size: 12px;
+    position: absolute;
+    top: 5%;
+    display: inline-block;
+    left: 10%;
+    background-color: var(--primary-red);
+    color: var(--white-only);
+}
+
+
+
+.card-with-dis .bat-badge-close {
+    border-radius: 50%;
+    font-size: 26px;
+    position: absolute;
+    display: inline-block;
+    top: 5%;
+    right: 10%;
+    color: var(--white-only);
+}
+
+.bat-badge-close {
+    display: none;
+}
+
+.bat-card i {
+    font-size: 1.25rem;
+}
+
+.bat-card button {
+    padding: 0.5rem 1rem;
+}
+
+.bat-card img {
+    max-width: 100%;
+    width: 300px;
```

Comment:
Why both max-width and width is used?

Reply:
Because, I wanted adjust it according to screen with % but also don't go beyond 300px

Reply:
Understood, then approved